lua/entities/gmod_wire_expression2/core/custom/sv_coroutines.lua

```diff
@@ -80,37 +83,24 @@ local function loadPrfData(instance,data)
 end
 
 local function createCoroutine(compiler,runtime,e2func)
-    local thread = coroutine.create(runtime)
+    local thread = coroutine_create(runtime)
     -- Data that we keep so we know whether a coroutine was created by e2 or not.
     compiler.coroutines[thread] = e2func
     return thread
 end
 
 local function runningCo(compiler) -- Don't return if a glua coroutine is running
-    local thread = coroutine.running()
+    local thread = coroutine_running()
     if not thread then return end
     if compiler.coroutines[thread] then return thread end
 end
 
-local function getE2FuncFromStr(compiler,funcname)
```

Comment:
- [x] Refactored this function and moved it into a single place, [click here to see](https://github.com/Vurv78/VExtensions/pull/15/files#diff-13513553927461b66e8fd31dbdf4979c1444325be4f59759e4217ca11732bd96R153).